modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/component.es.js

```diff
@@ -207,7 +206,7 @@ const component = function (id, value, componentConfig) {
 	if (arguments.length === 1) {
 		let component = components[id];
 
-		if (component && isFunction(component)) {
+		if (component && typeof component === 'function') {
```

Comment:
Probably don't need to race back and fix this, but `component &&` is redundant here.